.eslintrc.js

```diff
@@ -98,14 +104,14 @@ module.exports = {
         'no-multiple-empty-lines': 'off',
 
         // ORIGINAL tslint.json -> "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
-        'no-restricted-syntax': [
-          'error',
-          {
-            selector:
-              'CallExpression[callee.object.name="console"][callee.property.name=/^(debug|info|time|timeEnd|trace)$/]',
-            message: 'Unexpected property on console object was called',
-          },
-        ],
+        // 'no-restricted-syntax': [
```

Comment:
do we still need this code?

Reply:
I would recommend to use an logger instead of direct console output, as I think `console.{debug|info|warn|error|log|...}` is bad-practice like `System.{out|err}` in Java.

After a logger is integrated, this rule should be activated again. And also the `log` method should be included.